bin/rake

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
 begin
-  load File.expand_path("../spring", __FILE__)
+  load File.expand_path('../spring', __FILE__)
```

Comment:
@franzejr not sure if you used rubocop, but when I do I put `bin` dir in ignore because whenever a gem update the binstubs I will have to fix this again. Not a big deal, thought